<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>396</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>QRL8</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 39, 39);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="destinationFolderBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
background-color: rgba(255, 255, 255, 0);
border: 1px solid #bbb;
color: #bbb;
}
:hover{
border: 1px solid #ddd;
color: #ddd;
}
:pressed{
border: 1px solid white;
color: white;
}</string>
    </property>
    <property name="text">
     <string>Select Destination Folder...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataFileBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
background-color: rgba(255, 255, 255, 0);
border: 1px solid #bbb;
color: #bbb;
}
:hover{
border: 1px solid #ddd;
color: #ddd;
}
:pressed{
border: 1px solid white;
color: white;
}</string>
    </property>
    <property name="text">
     <string>Select Data File...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceFolderBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
background-color: rgba(255, 255, 255, 0);
border: 1px solid #bbb;
color: #bbb;
}
:hover{
border: 1px solid #ddd;
color: #ddd;
}
:pressed{
border: 1px solid white;
color: white;
}</string>
    </property>
    <property name="text">
     <string>Select Source Folder...</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 201, 3)</string>
    </property>
    <property name="text">
     <string>QRL8</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sourceFolderValidLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dataFileValidLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="destinationFolderValidLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="successLbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 255, 6);</string>
    </property>
    <property name="text">
     <string>Footage sorted successfully.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="goBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled,:disabled{
background-color: rgba(255, 255, 255, 0);
}
:enabled {
border: 1px solid rgb(33, 255, 6);
color: rgb(33, 255, 6);
}
:hover {
border: 1px solid rgb(102, 255, 102);
color: rgb(102, 255, 102);
}
:disabled {
border:  1px solid #555;
color: #888;
}</string>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <zorder>destinationFolderValidLbl</zorder>
   <zorder>dataFileValidLbl</zorder>
   <zorder>sourceFolderValidLbl</zorder>
   <zorder>destinationFolderBtn</zorder>
   <zorder>dataFileBtn</zorder>
   <zorder>sourceFolderBtn</zorder>
   <zorder>logo</zorder>
   <zorder>successLbl</zorder>
   <zorder>goBtn</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
